sdk/samples/core/mosquitto_multiple_rpc_sample.c

```diff
@@ -32,12 +32,6 @@ static const az_span server_client_id = AZ_SPAN_LITERAL_FROM_STR("controller-id"
 static const az_span content_type = AZ_SPAN_LITERAL_FROM_STR("application/json");
 static const az_span start_module_command_name = AZ_SPAN_LITERAL_FROM_STR("startModule");
 static const az_span stop_module_command_name = AZ_SPAN_LITERAL_FROM_STR("stopModule");
-static const az_span server_subscription_topic_format
-    = AZ_SPAN_LITERAL_FROM_STR("device/{executorId}/command/{name}");
-static const az_span client_subscription_topic_format
```

Comment:
Should we keep an example somewhere of how to specify custom topic formats?

Reply:
From the above comments it seems we want to use default topics for all our samples, it would be good to add this in our readme though